com.github.git-cola.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.github.git-cola
```

Comment:
also the app-id should be more like `com.github.git_cola.git-cola`

Reply:
That sounds horrible.

Reply:
It is what it is https://docs.flatpak.org/en/latest/conventions.html#application-ids